### PR TITLE
TT-5528 Note Category Graphic and color

### PR DESCRIPTION
**Pull request by gtryus**
- use note category graphic as default
- show color as border
- update reference with category
- put commas between search keywords

### CODE REVIEW

File: src/crud/useGraphicFind.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  return (recId: InitializedRecord, ref?: string) => {
+    let graphicRec = graphics.find(
+      (g) =>
+        g.attributes.resourceType === recId.type &&
+        g.attributes.resourceId === parseInt(recId?.keys?.remoteId ?? '0')
+    );
+    let color = undefined;
+    if (ref) {
+      const catText = ref.split(' ')[1];
+      const catRec = artifactCategory.find(
```
Comment (sarahentzel) [inline src/crud/useGraphicFind.ts:21]: The category showing in the reference should be localized, so would need to be unlocalized to find the category from the reference

File: src/crud/useSharedResUpdate.ts [hunk 2]
```diff
@@ -35,16 +39,10 @@ export const useSharedResUpdate = () => {
...
-              sharedResource.relationships.passage.data as RecordIdentity,
-              'reference',
-              `NOTE ${catRec.attributes.categoryname}`
-            )
-            .toOperation()
-        );
+      if (catRec && onUpdRef) {
+        const passage = sharedResource.relationships.passage
+          .data as RecordIdentity;
+        onUpdRef(passage.id, `NOTE ${catRec.attributes.categoryname}`);
```
Comment (sarahentzel) [inline src/crud/useSharedResUpdate.ts:45]: since the category name appears to the user, this should be localized

**Review by sarahentzel — CHANGES_REQUESTED** [review]
The category showing in the reference should be localized, so would need to be unlocalized to find the category from the reference

File: src/crud/useGraphicFind.ts (hunk 1 above)
Comment (gtryus) [reply src/crud/useGraphicFind.ts:21]: Good catch

File: src/crud/useSharedResUpdate.ts (hunk 2 above)
Comment (gtryus) [reply src/crud/useSharedResUpdate.ts:45]: yes, I have done this now.

**Pull request merged**